ch_5_AppDev_Intermediate/5_2_Speech_Recognition.ipynb

```diff
@@ -142,18 +130,18 @@
     "\n",
     "Once you have optimized the Whisper model using BigDL-LLM with INT4 optimization and loaded the Whisper processor, you are ready to begin transcribing the audio through model inference.\n",
     "\n",
-    "Let's start with the English audio file `audio_en.mp3`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:"
+    "Let's start with the English audio file `audio_en.wav`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:"
    ]
```

Comment:
I feel it maybe better to provide such `wget -O audio_en.mp3 https://datasets-server.huggingface.co/assets/common_voice/--/en/train/5/audio/audio.mp3` command to make how to get `audio_en.mp3` more clear.

Reply:
```shell
wget --no-check-certificate -O audio.wav https://datasets-server.huggingface.co/assets/carlot/AIShell/--/692ef58020d79b21f54eb25b15a4813d4f9650d7/--/default/train/84/audio/audio.wav
--2023-12-15 17:07:07--  https://datasets-server.huggingface.co/assets/carlot/AIShell/--/692ef58020d79b21f54eb25b15a4813d4f9650d7/--/default/train/84/audio/audio.wav
正在解析主机 child-prc.intel.com (child-prc.intel.com)... 10.239.120.56
正在连接 child-prc.intel.com (child-prc.intel.com)|10.239.120.56|:913... 已连接。
OpenSSL: error:0A000126:SSL routines::unexpected eof while reading
无法建立 SSL 连接。
```
When using `wget` command It is more likely to get network problem so I provided the link to download directly -_-， maybe I can add a note 'click the link to download the examples' or just leave it alone?